packages/ragbits-core/src/ragbits/core/utils/config_handling.py

```diff
@@ -13,39 +13,39 @@ class InvalidConfigError(Exception):
     """
 
 
-def get_cls_from_config(cls_path: str, default_module: ModuleType | None) -> Any:  # noqa: ANN401
+def import_by_path(path: str, default_module: ModuleType | None) -> Any:  # noqa: ANN401
```

Comment:
Note: those changes don't modify the behavior of this function - they just better describe the behavior that it had all along.